src/db/database.providers.ts

```diff
@@ -0,0 +1,19 @@
+import { createConnection } from 'typeorm';
+
+export const databaseProviders = [
+  {
+    provide: 'DATABASE_CONNECTION',
+    useFactory: async () => await createConnection({
+      type: 'postgres',
+      host: process.env.DB_HOST,
+      port: 5432,
+      username: process.env.DB_USER,
+      password: process.env.DB_PASS,
+      database: 'sl_test',
```

Comment:
In the real word all changeable parameters better to init via environment variables.
In this case,  _port_ & _database_ definitely should be initialized in this way.

Reply:
fixed